InvenTree/common/notifications.py

```diff
@@ -299,11 +299,12 @@ def trigger_notification(obj, category=None, obj_ref='pk', **kwargs):
             delivery_methods = (delivery_methods - IGNORED_NOTIFICATION_CLS)
 
         for method in delivery_methods:
-            logger.info(f"Triggering method '{method.METHOD_NAME}'")
+            logger.info(f"Triggering notification method '{method.METHOD_NAME}'")
             try:
                 deliver_notification(method, obj, category, targets, context)
             except NotImplementedError as error:
-                raise error
```

Comment:
@SchrodingersGat raising an error here was intentional - this way a wrongly implemented notification method sticks out directly and does not fail silently for a long time.
If you do not want this to raise errors I suggest at least showing a warning in the UI.

Reply:
Fair enough, I can see the merit in that. I'll make a new PR to revert this bit.